style/js/shield_defs.js

```diff
@@ -90,6 +90,22 @@ export function loadShields(shieldImages) {
     },
   };
 
+  var shield_us_us = [
+    shieldImages.shield40_us_us_2,
+    shieldImages.shield40_us_us_3,
+  ];
+  var padding_us_us = {
+    left: 2,
+    right: 2,
+    top: 4,
+    bottom: 5,
+  };
+  var usMUTCDNationalShield = {
```

Comment:
If you’re looking for a more convenient variable name, I think it’s called a “badge” shield.

Reply:
Good thinking; I'll go ahead and rename that.